src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -885,7 +896,7 @@
 
       .btn.btn-link {
         color: $clr-modal-close-color;
-        margin-top: 0.483333rem;
+        margin-top: 0.666667rem;
```

Comment:
Is this something we will use a lot in other places? Does it need to be a variable? `$clr-baselineRem_two_thirds` or `$clr-baselineRem_0666667`

Not a blocker.

Reply:
I believe @mathisscott will address things like this in his upcomign PR.

Reply:
Yep...